pkg/patterns/declarative/kustomize/disabled.go

```diff
@@ -0,0 +1,27 @@
+//go:build without_kustomize
+// +build without_kustomize
+
+package kustomize
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+func init() {
+	Kustomize = &SkipKustomize{}
+}
+
+var _ Kustomizer = &SkipKustomize{}
+
+// NoopKustomize ignore the `kustomization.yaml` file and won't run `kustomize build`.
+type SkipKustomize struct{}
+
+// Run is a no-op
+func (k *SkipKustomize) Run(ctx context.Context, _ filesys.FileSystem, _ string) ([]byte, error) {
+	log := log.FromContext(ctx)
+	log.WithValues("kustomizer", "SkipKustomize").Info("skip running kustomize")
+	return nil, nil
```

Comment:
I think you could return an error here, like `fmt.Errorf("kustomize support is not compiled in (built with tag without_kustomize)")`

Reply:
I like the error message. Updated